### PR TITLE
[Commands] Cleanup #xtargets Command.

### PR DESCRIPTION
**Pull request by Kinglykrab**
- Cleanup messages and logic.

### CODE REVIEW

File: zone/client.cpp [hunk 1]
```diff
@@ -7483,21 +7483,51 @@ void Client::RemoveAutoXTargets()
...
 		return;
+	}
+
+	auto xtarget_count = 0;
+
+	for (int i = 0; i < GetMaxXTargets(); ++i) {
+		c->Message(
+			Chat::White,
+			fmt::format(
+				"XTarget Slot {} | Type: {} ID: {} Name: {}",
```
Comment (Akkadius) [inline zone/client.cpp:7496]: It'd be nice to keep consistent debugging outputs

Eg

```
xtarget slot [{}]
```

vs

```
XTarget Slot {} |

Comment (Kinglykrab) [reply zone/client.cpp:7496]: All my other stuff is the same format as this. This isn’t a debug log, it’s a client message.

File: zone/client.cpp [hunk 2]
```diff
@@ -7483,21 +7483,51 @@ void Client::RemoveAutoXTargets()
...
 
-	for(int i = 0; i < GetMaxXTargets(); ++i)
-		c->Message(Chat::White, "Xtarget Slot: %i, Type: %2i, ID: %4i, Name: %s", i, XTargets[i].Type, XTargets[i].ID, XTargets[i].Name);
 	auto &list = GetXTargetAutoMgr()->get_list();
 	 // yeah, I kept having to do something for debugging to tell if managers were the same object or not :P
 	 // so lets use the address as an "ID"
-	c->Message(Chat::White, "XTargetAutoMgr ID %p size %d", GetXTargetAutoMgr(), list.size());
+	c->Message(
+		Chat::White,
+		"XTargetAutoMgr ID %p size %d",
```
[the remark this answers is not in the document]
Comment (Akkadius) [reply zone/client.cpp:7512]: To print pointer addresses is to use `fmt::ptr`

eg

```
std::string s = fmt::format("{},{}", fmt::ptr(pAccels), count);
```

File: zone/client.cpp (hunk 1 above)
Comment (Akkadius) [reply zone/client.cpp:7496]: It's a client message used in the same sense, it's printing variables used for debugging

**Pull request merged**